src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,15 @@
+const ArticleCard = ({ article }) => {
+    // console.log(article.article_id);
+    
+  return (
+    <div className="article-card">
+      <li>
+        <a href={`/articles/${article.topic}/${article.article_id}`}>
+          <h3>{article.title}</h3>
+        </a>
+      </li>
+    </div>
+  );
```

Comment:
This does not follow the plan either as it is just loading a single title and nothing more. You dont need to finish every ticket looking perfect, but I would recommend using all available content and displaying this on the page at the very least.

Reply:
The original plan has images, but this data (the array with all articles) doesn't contain the article images, hence why it looks different. The only data in this is article_id, author, comment_count, created_at, title, topic, and votes.